### PR TITLE
Feature #6 tag selection for drag and drop element

### PR DESCRIPTION
**Pull request by akubasa**
This is the implementation for tag selection in Drag and Drop elements.
Tested and already working with it.

### CODE REVIEW

**Comment by cameronhimself** [comment]
Thanks so much for doing this!

File: src/Drop.vue [hunk 1]
```diff
@@ -16,8 +16,11 @@
 	const insideElements = new Set();
 
 	export default {
+        props: {
+            tag: { type: String, default: 'div' },
+        },
```
Comment (cameronhimself) [inline src/Drop.vue]: Please use tabs for indentation.

File: README.md [hunk 2]
```diff
@@ -126,6 +126,11 @@ By default, a custom drag image is position so that its top-left corner is ancho
 **default**: `true`  
 If the `Drag` `image` slot is used, toggle whether or not the HTML is rendered off-screen. See the `image` slot documentation for more details.
 
+### `tag`
+**validation** `String`
+**default**: `div`
+Drag or Drop element's wrapper , defaults to div
```
Comment (cameronhimself) [inline README.md]: Can you add a period to the end, and remove the space before the comma?

File: src/Drop.vue [hunk 3]
```diff
@@ -16,8 +16,11 @@
...
 		data: () => ({ dataKey: null }),
-		computed: {
+        computed: {
```
Comment (cameronhimself) [inline src/Drop.vue]: It doesn't look like this indentation should be changed.

**Comment by akubasa** [comment]
@cameronhimself, done

**Comment by cameronhimself** [comment]
I'll publish to npm this weekend. Thanks again!

**Comment by akubasa** [comment]
Hello,
I cant see a released version on npm yet. When will you release to npm?
-------- Ursprüngliche Nachricht --------Von: "J. Cameron McDonald" <notifications@github.com> Datum: 17.11.17  21:20  (GMT+01:00) An: cameronhimself/vue-drag-drop <vue-drag-drop@noreply.github.com> Cc: akubasa <a.kubasa@at-visions.com>, Author <author@noreply.github.com> Betreff: {Disarmed} Re: [cameronhimself/vue-drag-drop] Feature #6 tag selection for drag and drop element (#10) 
I'll publish to npm this weekend. Thanks again!

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/cameronhimself/vue-drag-drop","title":"cameronhimself/vue-drag-drop","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/cameronhimself/vue-drag-drop"}},"updates":{"snippets":[{"icon":"PERSON","message":"@cameronhimself in #10: I'll publish to npm this weekend. Thanks again!"}],"action":{"name":"View Pull Request","url":"https://github.com/cameronhimself/vue-drag-drop/pull/10#issuecomment-345355435"}}}

**Comment by cameronhimself** [comment]
I'm sorry, busy weekend. I'll release tonight after I get off work.

**Comment by cameronhimself** [comment]
Released! Thanks again, and sorry for the delay.

**Comment by akubasa** [comment]
I say thank you!
-------- Ursprüngliche Nachricht --------Von: "J. Cameron McDonald" <notifications@github.com> Datum: 22.11.17  00:02  (GMT+01:00) An: cameronhimself/vue-drag-drop <vue-drag-drop@noreply.github.com> Cc: akubasa <a.kubasa@at-visions.com>, Author <author@noreply.github.com> Betreff: {Disarmed} Re: [cameronhimself/vue-drag-drop] Feature #6 tag selection for drag and drop element (#10) 
Released! Thanks again, and sorry for the delay.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.


  
  




{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/cameronhimself/vue-drag-drop","title":"cameronhimself/vue-drag-drop","subtitle":"GitHub repository","main_image_url":"https://cloud.githubusercontent.com/assets/143418/17495839/a5054eac-5d88-11e6-95fc-7290892c7bb5.png","avatar_image_url":"https://cloud.githubusercontent.com/assets/143418/15842166/7c72db34-2c0b-11e6-9aed-b52498112777.png","action":{"name":"Open in GitHub","url":"https://github.com/cameronhimself/vue-drag-drop"}},"updates":{"snippets":[{"icon":"PERSON","message":"@cameronhimself in #10: Released! Thanks again, and sorry for the delay."}],"action":{"name":"View Pull Request","url":"https://github.com/cameronhimself/vue-drag-drop/pull/10#issuecomment-346189924"}}}

**Pull request merged**